### DISCUSSION TITLE
After changing to monorepo cant find project

### DISCUSSION THREAD

**Discussion by Mayb3Nots** [codemagic-ci-cd/codemagic-docs#2105] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

1.Have a monorepo using Melos
2. Go into Codemagic dashboard and try to scan project path. Only shows root. Which is the pubspec.yaml of the Melos project

### Expected results

When scanned for project it should show my other nested projects

### Actual results

No projects found only root

**Comment by himesh-cm** [comment]
Hi @Mayb3Nots,
The `codemagic.yaml` file must be created in the root directory of the repository. By default, the working directory is the root of the repository.

By defining working_directory, every command that is defined in the scripts section will run inside that particular directory which is “my_first_app” in our sample above.

For example:

```
workflows:
  my-first-app-workflow:
    name: Drivers app workflow
    working_directory: my_first_app
```

A sample project can be found [here](https://github.com/codemagic-ci-cd/codemagic-sample-projects/blob/main/codemagic.yaml).

**Reply by Mayb3Nots** [reply]
Hi @himesh-cm I am not using ```codemagic.yaml``` I am following this article https://blog.codemagic.io/codemagic-monorepo/ which seems like I don't need to use ```codemagic.yaml``` but I cant seem to get past the step where it detected multiple project path after rescan.

**Reply by himesh-cm** [reply]
Please share a team id so that i can look into the issue.

**Reply by Mayb3Nots** [reply]
Where can I find this team ID? I am not using a team though

**Reply by himesh-cm** [reply]
Please navigate to `Teams > Settings > Personal` and share the URL in the following format: `https://codemagic.io/teams/xxxxxxxxxx`.

**Reply by Mayb3Nots** [reply]
6104e51ef16add6466aa0dee

**Reply by himesh-cm** [reply]
You have multiple apps. Could you please share the names of the apps so I can investigate?

**Reply by Mayb3Nots** [reply]
It is Smap

**Reply by Mayb3Nots** [reply, marked as answer]
Hi I have removed the .git folder for my monorepos and now they are detected
